server/src/main/java/org/apache/iotdb/db/mpp/sql/statement/crud/InsertMultiTabletsStatement.java

```diff
@@ -64,6 +64,14 @@ public List<TSDataType[]> getDataTypesList() {
     return dataTypesList;
   }
 
+  public List<Boolean> getAlignedList() {
```

Comment:
Why we don't have this function in other `insertNodes`?

Reply:
Used for batch insert for inserting into different devices

Reply:
It seems that only 'statement' has this function and 'node' does not. Why is this?

Reply:
Batch insert statement needs this function to collect all aligned info of all devices to automatically create time series before creating insert node. We don't need to collect such info afterwards.

Reply:
OK